OutOfSchool/OutOfSchool.WebApi/Services/SubordinationStructure/InstitutionHierarchyService.cs

```diff
@@ -102,6 +107,17 @@ public async Task<List<InstitutionHierarchyDto>> GetChildren(Guid? parentId)
         {
             logger.LogInformation("Getting all children InstitutionHierarchies started.");
 
+            string cacheKey = $"InstitutionHierarchyService_GetChildren_{parentId}";
```

Comment:
Maybe we will need to save such cacheKey templates in one place and then use `string.Format` for retrieving string we want.